tests/test_examplegen.py

```diff
@@ -51,7 +50,11 @@ def relative_endpoints_to_file_annotation_fixture(
       end=datetime.timedelta(seconds=end_seconds),
       label='Oo',
   )
-  return annotation.relative_endpoints(clip_metadata)
+  clip_annotation = annotation.make_relative(clip_metadata)
+  if clip_annotation:
+    return clip_annotation.begin, clip_annotation.end
```

Comment:
Would it be simpler to return the Optional[ClipAnnotation] instead of handling the None condition twice?

Reply:
Done.